examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -3,7 +3,7 @@
 ===============================================================
 
 This example does a simple <JUNK TET FOR TESTING TO BE REMOVED LATER>
-but efficient decoding on the Haxby dataset:
+JUNK but efficient decoding on the Haxby dataset:
```

Comment:
Is this how you're testing the CI infrastructure ? :rofl: I'm a little nervous this will get left here, so adding a comment just to flag it :smile_cat:

Reply:
Haha Yeah but I will go through the changed files and remove them, you adding a comment absolutely adds another event in the memory, making it unlikely I''ll forget this. 😄